test/testdrive/avro-decode-mismatched-columns.td

```diff
@@ -65,8 +65,5 @@ $ kafka-ingest format=avro topic=decode-2to1 schema=${2columns-nodefault} timest
   FORMAT AVRO USING SCHEMA '${1column}'
   ENVELOPE NONE
 
-> SELECT * FROM decode_2to1
-f1
----
-123
-456
+! SELECT * FROM decode_2to1
+Unexpected bytes remaining
```

Comment:
Is this better? Why?

Reply:
Yes -- previously this was checking that Materialize didn't crash (but returned gibberish answers) for a particular combination of bad schemas; now we (correctly) return an error.